packages/fluentui/react-northstar-fela-renderer/src/felaDisableAnimationsPlugin.ts

```diff
@@ -0,0 +1,51 @@
+import { ICSSInJSStyle, RendererParam } from '@fluentui/styles';
+import { Renderer } from './types';
+
+const animationProps: (keyof ICSSInJSStyle)[] = [
+  'animation',
+  'animationName',
+  'animationDuration',
+  'animationTimingFunction',
+  'animationDelay',
+  'animationIterationCount',
+  'animationDirection',
+  'animationFillMode',
+  'animationPlayState',
+];
+
+/**
+ * Fela plugin for disabling animations. The animations are disabled or not based on the
+ * props' disableAnimations param. If the value of the prop is true, all animation related
+ * styles are removed.
+ *
+ * Caution! Infinite recursion is possible in case if style object has links to self in the props
+ * tree.
+ */
+const felaDisableAnimationsPlugin = (
```

Comment:
All plugins received a small refactor:
- were simplified where options are not required
- typings improved

Actually the main reason to do it: it's avoid fighting with typings.